doc/release-notes/0.12/0.12.1.md

```diff
@@ -0,0 +1,151 @@
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.12.1 release notes
+
+These release notes support the [Eclipse OpenJ9 0.12.1 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.12.1/plan).
+
+
+## Binaries and supported environments
+
+OpenJ9 release 0.12.1 supports OpenJDK 8 and OpenJDK 11. Binaries are available at the AdoptOpenJDK project:
+
+- [OpenJDK with OpenJ9 version 8](https://adoptopenjdk.net/archive.html?variant=openjdk8&jvmVariant=openj9)
+- [OpenJDK with OpenJ9 version 11](https://adoptopenjdk.net/archive.html?variant=openjdk11&jvmVariant=openj9)
+
+All builds are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests at AdoptOpenJDK.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html).
+
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.12.1. Further information about these changes can be found in the [user documentation](https://www.eclipse.org/openj9/docs/version0.12.1/).
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/4530">#4530</a></td>
+<td valign="top">OpenSSL acceleration of the Digest algorithm is disabled by default.</td>
+<td valign="top">OpenJDK8 and later (All platforms)</td>
+<td valign="top">Intermittent segmentation problems are observed when accelerating the Digest algorithm. This algorithm is now
+disabled by default. The <b>-Djdk.nativeDigest</b> system property no longer has any effect. </td>
+</tr>
+
+
+</table>
+
+
+## Known Issues
+
+The v0.12.0 release contains the following known issues and limitations:
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1">
+<thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Platform</th>
+<th valign="bottom">Impact</th>
+<th valign="bottom">Workaround</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/4530">#4530</a></td>
+<td valign="top">Intermittent segmentation faults occur when using OpenSSL V1.1.x acceleration for the Digest algorithm</td>
+<td valign="top">All platforms</td>
+<td valign="top">You cannot use cryptographic acceleration for the Digest algorithm.</td>
+<td valign="top">Use an alternative algorithm. Currently, Digest is disabled by default.</td>
```

Comment:
Do we need #4530 listed twice?
Re `Use an alternative algorithm.` this is confusing. The user doesn't need to do anything since the openssl digest support is disabled by default.

Reply:
`You cannot use cryptographic acceleration for the Digest algorithm.` sounds like it may be a concerning problem, while the estimated the acceleration was only worth about ~5% perf and unlikely to be noticeable.

Reply:
>Do we need #4530 listed twice?

It is both a change and an issue (regression). I figured it deserved a mention in both places.

>Re Use an alternative algorithm. this is confusing. The user doesn't need to do anything since the openssl digest support is disabled by default.

I was guessing users might want cryptographic acceleration but I'll remove it

> You cannot use cryptographic acceleration for the Digest algorithm. sounds like it may be a concerning problem, while we estimated the acceleration was only worth about ~5% perf and unlikely to be noticeable.

Really? Wow. If noone was likely to be bothered about it, I guess it is a regression that can be ignored. In which case I'll remove it from the issues table completely.